cpp/csv_utils_module/algorithm/csv_utils.cpp

```diff
@@ -14,7 +14,7 @@ void CreateCsvFile(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result
 
     std::ofstream fout;
     fout.open(std::string(filepath), is_append ? std::ofstream::app : std::ofstream::out);
-    fout << content << std::endl;
+    fout << content;
```

Comment:
Do you need to use `std::flush` here? (https://en.cppreference.com/w/cpp/io/manip/flush) or `fout.close()` will flush the buffer?

Reply:
Not sure actually :man_shrugging: 
I guess I do since `std::endl` actually calls `std::flush`.

Reply:
From cpp ref on `ofstream` [link](https://en.cppreference.com/w/cpp/io/basic_ofstream) we can see following:
- `fout`'s destructor will actually call close: https://en.cppreference.com/w/cpp/io/basic_ofstream/close
- close will call implicit flush via the following function: https://en.cppreference.com/w/cpp/io/basic_filebuf/close

So is unnecessary, and close too.

Double-checked here:
https://stackoverflow.com/questions/16268433/what-does-stdofstreamclose-actually-do

I trust cpp ref, but since this is a really sensitive matter, if I failed to understand some case here, this might be really difficult to debug issue, so would leave it as it is.

Reply:
I included `std::flush` just in case then. (as a replacement for endl because that also flushes)